airflow/contrib/operators/content_email_components.py

```diff
@@ -0,0 +1,75 @@
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+import pandas as pd
+import subprocess
+import os
+import logging
+
+def dataTable(data,dim_name, metric_name):
+	#infer which range of dates is available for width of table
+	date_list = []
+	date_list_formatted = []
+	for row in data:
+		if row['ds'] in date_list:
+			break
+		else:
+			date_list.append(row['ds'])
+			if row['ds'][5] == '0':
+				date_list_formatted.append(row['ds'][6:])
+			else:
+				date_list_formatted.append(row['ds'][5:])
+	
+	cnum = 0 #column number
+	width = len(date_list)
+	output = '''<table 
```

Comment:
Tried it and preferred more granular control. (especially filling in blanks as well) My idea was that it would be easy to add different python callables that generate either html or images.